packages/envied/README.md

```diff
@@ -148,8 +148,11 @@ Add the `obfuscate` flag to EnviedField
 
 ```dart
 @EnviedField(obfuscate: true)
+static final String key = _Env.key;
 ```
 
+Note: In this case you should use `static final` instead of `static const`.
```

Comment:
Not sure why this is needed as line 151 already explains it clearly.

Reply:
Because it makes the difference more evident, over explaining is never too bad IMHO.

Reply:
Please remove it.